tests/test_C0_data_processing.py

```diff
@@ -1131,6 +1132,26 @@ def test_compute_timeseries_properties_TIMESERIES_not_in_dict_asset():
     ), f"The function has changed the dict_asset to {dict_asset}, eventhough it should not have been modified and stayed identical to {dict_exp}."
 
 
+def test_compute_timeseries_properties_TIMESERIES_not_in_dict_asset():
+    dict_asset = {
+        UNIT: "str",
+        LABEL: "str",
+    }
+    dict_exp = copy.deepcopy(dict_asset)
+    assert (
+        dict_exp == dict_asset
+    ), f"The function has changed the dict_asset to {dict_asset}, eventhough it should not have been modified and stayed identical to {dict_exp}."
+
+
+def test_replace_nans_in_timeseries_with_0():
+    timeseries = pd.Series([10, np.nan, 100, 150, 200, np.nan, 91])
+
+    timeseries = C0.replace_nans_in_timeseries_with_0(timeseries, "any")
+
+    exp = pd.Series([10, 0, 100, 150, 200, 0, 91])
+    assert (timeseries == exp).all(), f"The timeseries is improperly changed."
+
+
```

Comment:
You could also assert that the logging.error was issued. See [E4.test_detect_excessive_excess_generation_in_bus_warning_is_logged](https://github.com/rl-institut/multi-vector-simulator/blob/e927ae93ea8ea1857ef36026d36cb52daad1bc32/tests/test_E4_verification.py#L116) for inspiration (`caplog` is a magic pytest builtin parameter, you don't need to import anything to use it)